charts/aws-config-generation/values.yaml

```diff
@@ -0,0 +1,86 @@
+project: si
+owner: infra-eng@chanzuckerberg.com
+env: playground
```

Comment:
We don't want these to be defaults for the chart

Reply:
included by accident, removed